### PR TITLE
Docs: Update documentation-style-guide.md

### PR DESCRIPTION
**Pull request by oddlittlebird**
Added Commands and code section.

**Special notes for your reviewer**: Related to PR #21314

### CODE REVIEW

File: contribute/style-guides/documentation-style-guide.md [hunk 1]
```diff
@@ -81,7 +81,18 @@ Always give the reader some idea of what to expect in the reference. Avoid blind
 When possible, use the exact title of the page or section you are linking to as the link text.
 
 **Example**
-* Refer to the [Documentation style guide](documentation-style-guide.md) for information about word usage and capitalization guidelines.
+Refer to the [Documentation style guide](documentation-style-guide.md) for information about word usage and capitalization guidelines.
+
+### Commands and codes
```
Comment (ryantxu) [inline contribute/style-guides/documentation-style-guide.md]: what is "codes"?  

Maybe "Command line examples"?

Comment (oddlittlebird) [reply contribute/style-guides/documentation-style-guide.md]: Ooh, much better! I was to convey something like "stuff in code blocks," but I wanted to be more specific than that. Applied your change.

File: contribute/style-guides/documentation-style-guide.md [hunk 2]
```diff
@@ -81,7 +81,18 @@ Always give the reader some idea of what to expect in the reference. Avoid blind
...
 
 **Example**
-* Refer to the [Documentation style guide](documentation-style-guide.md) for information about word usage and capitalization guidelines.
+Refer to the [Documentation style guide](documentation-style-guide.md) for information about word usage and capitalization guidelines.
+
+### Commands line examples
+
+* Do not assume everyone is using Linux. Make sure instructions include enough information for Windows and Mac users to successfully complete procedures.
+
+* Do not add `$` before commands. Make it easy for users to copy and paste commands.
```
Comment (jjasghar) [inline contribute/style-guides/documentation-style-guide.md]: :100: NICE. Thank you, it's always good to _read_ the commands, but like most I know I love my copy :spaghetti:.

File: contribute/style-guides/documentation-style-guide.md [hunk 3]
```diff
@@ -81,7 +81,18 @@ Always give the reader some idea of what to expect in the reference. Avoid blind
...
+
+  * **Wrong:** `$ sudo yum install grafana`
+  * **Right:** `sudo yum install grafana`
```
Comment (marefr) [inline contribute/style-guides/documentation-style-guide.md]: Maybe good to add something about code examples. Maybe here or separate section?

For terminal examples and Grafana configuration we use bash
```bash
sudo yum install grafana`
```

For HTTP request/response we use http:
```http
GET /api/dashboards/id/1/permissions HTTP/1.1
Accept: application/json
Content-Type: application/json
Authorization: Bearer eyJrIjoiT0tTcG1pUlY2RnVKZTFVaDFsNFZXdE9ZWmNrMkZYbk
```

Comment (oddlittlebird) [reply contribute/style-guides/documentation-style-guide.md]: Good idea. Done.

File: contribute/style-guides/documentation-style-guide.md [hunk 4]
```diff
@@ -81,7 +81,30 @@ Always give the reader some idea of what to expect in the reference. Avoid blind
 When possible, use the exact title of the page or section you are linking to as the link text.
 
 **Example**
-* Refer to the [Documentation style guide](documentation-style-guide.md) for information about word usage and capitalization guidelines.
+Refer to the [Documentation style guide](documentation-style-guide.md) for information about word usage and capitalization guidelines.
+
+### Commands line examples
```
Comment (marefr) [inline contribute/style-guides/documentation-style-guide.md]: Nit, Command line examples?

**Review by marefr — APPROVED** [review]
LGTM, left a minor comment

File: contribute/style-guides/documentation-style-guide.md (hunk 4 above)
Comment (oddlittlebird) [reply contribute/style-guides/documentation-style-guide.md]: Thought I had fixed this. One moment.

**Pull request merged**